FirebaseInstallations/Source/Library/FIRSecureStorage.h

```diff
@@ -60,6 +60,12 @@ NS_ASSUME_NONNULL_BEGIN
 - (FBLPromise<NSNull *> *)removeObjectForKey:(NSString *)key
                                  accessGroup:(nullable NSString *)accessGroup;
 
+#if TARGET_OS_OSX
+/// If not `nil`, then only this keychain will be used to save and read data (see
+/// `kSecMatchSearchList` and `kSecUseKeychain`. It is mostly intended to be used by unit tests.
```

Comment:
Delete "mostly" since it is in `#if TARGET_OS_OSX`

Reply:
Technically it still can be used in production on mac os.

Reply:
Oh I see. I misunderstood when I read it.

If it's only for tests, why not add as a category in the tests?

Reply:
Do you mean declare the property in the class extension so it is not exposed unless tests define a category? Not sure if we will benefit from it much. I think, since the class is private anyway, it should be fine to keep the property in the header. Also, if this class becomes public, then having this property also makes sense because it can be actually used for a real application if needed.

Reply:
@paulb777  I am going to merge the PR, but feel free to respond to the comment if you still have concerns. I'll update it in next PRs if needed.